app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
```

Comment:
- [ ] I must commend your effort so far 👍🏽 . I think your views for your devise registration is still not making the wireframe design guideline given. 

I think you should consider centering the entire form as a whole(e.g use flexbox} as suggested by the previous reviewer, taking particular on the width and height of the input fields, font size of texts, and the position of the submit button. It will be nice to improve the overall user experience of your app 


your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/233491386-5d339ced-6a13-4898-8207-6178a991b606.png)

wireframe 👇🏽 
let your registration styling also look like this login wireframe
![image](https://user-images.githubusercontent.com/63473584/233491215-cda1a707-9756-4db6-98fa-f85ddf9d4631.png)